src/site/content/en/blog/preload-optional-fonts/index.md

```diff
@@ -0,0 +1,110 @@
+  <figcaption class="w-figcaption">Previous <code>font-display: optional</code> behavior in Chrome when font is downloaded <b>after</b> the 100ms block period</figcaption>
+</figure>
+
+However, in the case that the font is downloaded before the 100ms block period completes, the custom
+font is rendered and used on the page.
+
+<figure class="w-figure">
+  <img src="./previous-behavior-pass.png" alt="Diagram showing previous optional font behavior when font loads in time">
+  <figcaption class="w-figcaption">Previous <code>font-display: optional</code> behavior in Chrome when font is downloaded <b>before</b> the 100ms block period</figcaption>
+</figure>
+
+Chrome re-renders the page **twice** in both instances, regardless of whether the fallback font
+is used or if the custom font finishes loading in time. This causes a slight flicker of invisible
+text and, in cases when a new font is rendered, layout jank that moves some of the page's content.
+This occurs even if the font is stored in the browser's disk cache and can load well before the
+block period is complete.
+
+Optimizations have landed In Chrome 82 to entirely remove the first render cycle for optional fonts
+that are preloaded with [`<link rel="preload'>`](https://web.dev/codelab-preload-web-fonts/).
+Instead, rendering is blocked until the custom font has finished loading or a certain period of time
+has passed. This timeout period is currently set at 100ms, but may possibly change in the near
+future to optimize performance.
+
+<figure class="w-figure">
+  <img src="./new-behavior-fail.png" alt="Diagram showing new preloaded optional font behavior when font fails to load">
+  <figcaption class="w-figcaption">New <code>font-display: optional</code> behavior in Chrome when fonts are preloaded and font is downloaded <b>after</b> the 100ms block period (no flash of invisible text)</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="./new-behavior-pass.png" alt="Diagram showing new preloaded optional font behavior when font loads in time">
+  <figcaption class="w-figcaption">New <code>font-display: optional</code> behavior in Chrome when fonts are preloaded and font is downloaded <b>before</b> the 100ms block period (no flash of invisible text)</figcaption>
+</figure>
+
+Preloading optional fonts in Chrome removes the possibility of layout jank and flash of unstyled
+text. This matches the required behavior as specified in [CSS Fonts Module Level
+4](https://drafts.csswg.org/css-fonts-4/#valdef-font-face-font-display-optional) where optional
+fonts should never cause re-layout and user agents can instead delay rendering for a suitable period
+of time.
+
+Although it is not necessary to preload an optional font, it greatly improves the chance for it to
+load before the first render cycle, especially if it is not yet stored in the browser's
+cache.
+
+## Conclusion
+
+We're interested to hear your experiences preloading optional fonts with these new optimizations in
```

Comment:
```suggestion
The Chrome team is interested to hear your experiences preloading optional fonts with these new optimizations in
```
Avoid we because it implies that web.dev's opinion == Chrome's opinion